app/src/main/java/com/kickstarter/libs/utils/extensions/ProjectExt.kt

```diff
@@ -52,6 +52,8 @@ fun Project.updateProjectWith(config: Config, user: User?): Project {
         .build()
 }
 
+fun Project.showLatePledgeFlow() = this.isInPostCampaignPledgingPhase() ?: false && this.postCampaignPledgingEnabled() ?: false
```

Comment:
@mtgriego ack for this piece, they both need to be true, to show latePledges flow within the flow

Reply:
yeah, dont know why it wasn't just one flag, maybe so it can be enabled but not allowed for some projects, but on our end one boolean would suffice